manifests/tools/memtest.pp

```diff
@@ -8,11 +8,18 @@
 
   $tftp_root = $pxe::tftp_root
 
-  exec { 'retrieve memtest image':
-    path    => ['/usr/bin', '/usr/local/bin', '/bin'],
-    command => "wget -q -O - ${url} | gunzip > ${tftp_root}/tools/memtest86+",
-    creates => "${tftp_root}/tools/memtest86+",
-    require => Class['Pxe::Tools'],
+  archive { "${tftp_root}/tools/memtest86+-4.20.bin.gz":
```

Comment:
We've got a minimal example in https://github.com/voxpupuli/puppet-pxe/blob/master/spec/classes/init_spec.rb for the init class. Can you add the same for the pxe::tools::memtest class to ensure it still compiles? Same for the pxe::syslinux::direct class.